### ISSUE TITLE
Code suggestions containing `{}` are escaped in log output

### ISSUE THREAD

**Issue by PabstMirror** [BrettMayson/HEMTT#643]
```
31 │ if (({_x isKindOf "house"} count _intersects) > 0) then { true } else { false };
   │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use select
   = try: [false, true] select ( count _intersects > 0)
```

**Comment by BrettMayson** [comment]
Might be fixed from unrelated changes while debugging SQFC, will need to recheck with 1.11.2

**Comment by PabstMirror** [comment]
now `try: [false, true] select (_x isKindOf "house" count _intersects > 0)`
which is better but still missing the `{`

**Comment by BrettMayson** [comment]
Fixed in 70a1c81fa3b8cf714d76c3305a47ea5ae79b9b38, will be in the next release
